stdlib-integration-tests/task/tests/integration-tests/scheduler_stop_test.bal

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/runtime;
+import ballerina/task;
+import ballerina/test;
+
+const SUCCESS_RESPONSE = "Task successfully stopped";
+const FAILURE_RESPONSE = "Stopping the task failed";
+
+listener http:Listener timerStopListener = new (15006);
+
+int count = 0;
+
+service timerService = service {
+    resource function onTrigger() {
+        count = count + 1;
+    }
+};
+
+@http:ServiceConfig {
+    basePath: "/"
+}
+service TimerStopResultService on timerStopListener {
```

Comment:
We don't need this either.

Reply:
This testcase has been moved to the task module unit test